.github/workflows/conda_and_docs_build.yml

```
@@ -54,7 +55,7 @@ jobs:
         ARTIFACT_NAME: 'DocumentationHTML'
         PACKAGE_FOLDER_PATH: 'conda_package'
         PACKAGE_NAME: 'cil'
-        PACKAGE_CONDA_CHANNELS: 'conda-forge -c intel -c astra-toolbox/label/dev -c cvxgrp -c ccpi'
+        PACKAGE_CONDA_CHANNELS: '-c conda-forge -c intel -c astra-toolbox/label/dev -c cvxgrp -c ccpi'
```

```suggestion
        PACKAGE_CONDA_CHANNELS: 'conda-forge -c intel -c astra-toolbox/label/dev -c cvxgrp -c ccpi'
```
The sphinx action already has a -c in front of the first channel by default